app/pb_kits/playbook/pb_currency/currency.rb

```diff
@@ -3,14 +3,15 @@
 module Playbook
   module PbCurrency
     class Currency < Playbook::PbKit::Base
+      include ActionView::Helpers::NumberHelper
       PROPS = %i[configured_align
                  configured_classname
                  configured_data
                  configured_id
                  configured_label
                  configured_value
```

Comment:
Instead of calling this `value`, can we call this `amount`? Value seems pretty generic.